BuildLoader.py

```diff
@@ -191,6 +191,9 @@ def __init__(self, *args, **kwargs):
 		self.ENABLE_SMBIOS         = 0
 		self.ENABLE_LINUX_PAYLOAD  = 0
 		self.ENABLE_CONTAINER_BOOT = 1
+		self.ENABLE_CSME_UPDATE    = 0
+
+		self.BUILD_CSME_UPDATE_DRIVER    = 0
```

Comment:
Can we enable this on QEMU by default even if actual validation is not available?